Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/ViewModel/WPFBoardCommunication.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NoughtsAndCrossesWPF
+{
+    class WPFBoardCommunication : IBoardCommunication
+    {
+
+        public Position GetPlayerInput(object coordinatesRepresentation)
+        {
+            int oneDimensionalPosition = Convert.ToInt32(coordinatesRepresentation);
+            Position twoDimensionalPosition = TranslateOneDimensionalPosition(oneDimensionalPosition);
+            return twoDimensionalPosition;
+        }
+
+        public Position TranslateOneDimensionalPosition(int oneDimensionalPosition)
+        {
+            int vertical = oneDimensionalPosition / 3;
+            int horizontal = oneDimensionalPosition - (vertical * 3);
+            return new Position(horizontal, vertical);
+        }
+
+        public void ShowBoard(GameBoard board)
+        {
+            //translateBoardToOneDimensional(board);
```

Comment:
Show* methods probably belong to ViewModel. This class should be used by ViewModel, not Model. It translates position between 2D and 1D just because it's easier to use in WPF. Models don't care about that stuff usually.